translate/v23tov30/v23tov30.go

```diff
@@ -233,9 +233,16 @@ func translateGroups(groups []old.PasswdGroup) (ret []types.PasswdGroup) {
 
 func translateUnits(units []old.Unit) (ret []types.Unit) {
 	for _, u := range units {
+		var boolPtr *bool
```

Comment:
Let's call this `enabled`, rather than its type.

A comment like `// Follows logic in Ignition createUnits()` wouldn't hurt either.